elasticluster/share/playbooks/roles/jupyter/tasks/python.yml

```
@@ -54,6 +52,17 @@
     state: '{{ pkg_install_state }}'
   loop: '{{jupyter_additional_os_packages[ansible_os_family]}}'
 
+- name: Install mysql development package.
+  package:
+    name: 'mysqlclient-dev'
+    state: '{{pkg_install_state}}'
+  when: is_ubuntu_15_10_or_later
```

I think this would work on Ubuntu 14.04 as well.
```suggestion
  when: is_ubuntu_14_04_or_later
```